packages/trader/src/AppV2/Components/RiskManagementItem/__tests__/risk-management-item.spec.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import RiskManagementItem from '../risk-management-item';
+import useContractDetails from 'AppV2/Hooks/useContractDetails';
+import { CONTRACT_TYPES } from '@deriv/shared';
+
+jest.mock('@deriv/translations', () => ({
+    Localize: ({ i18n_default_text }: { i18n_default_text: string }) => <span>{i18n_default_text}</span>,
+    localize: (text: string) => text,
+}));
+
+jest.mock('@deriv-com/quill-ui', () => ({
+    ActionSheet: {
+        Root: ({ children }: { children: React.ReactNode }) => <div>{children}</div>,
+        Portal: ({ children }: { children: React.ReactNode }) => <div>{children}</div>,
+        Header: () => <div>Action Sheet Title</div>,
+        Content: ({ children }: { children: React.ReactNode }) => <div>{children}</div>,
+        Footer: ({ primaryAction }: { primaryAction: { content: React.ReactNode; onAction: () => void } }) => (
+            <button onClick={primaryAction.onAction}>{primaryAction.content}</button>
+        ),
+    },
+    Text: ({ children, size, color }: { children: React.ReactNode; size: string; color?: string }) => (
```

Comment:
Do we need to mock all quill components? Btw you are not using `size` prop

Reply:
We are not testing them here, so why wouldn't we mock it? If tomorrow one of these basic components breaks. I don't want to fix these unrelated tests multiple times.